nixos/modules/misc/ids.nix

```diff
@@ -307,6 +307,7 @@
       duplicati = 289;
       monetdb = 290;
       restic = 291;
+      hledger = 292;
```

Comment:
Since this service only needs a single file, we don't need fixed uids for it. Please use remove them here so dynamic ids will be allocated when the service is enabled.